### DISCUSSION TITLE
Max. value of temperature in heat equation example

### DISCUSSION THREAD

**Discussion by adtzlr** [Ferrite-FEM/Ferrite.jl#1026] [Q&A] [answered]
Hi,

shouldn't the max. value of the [heat equation example](https://ferrite-fem.github.io/Ferrite.jl/dev/tutorials/heat_equation/) be ≈0.073 instead of 0.295? [[1](https://scikit-fem.readthedocs.io/)]

![image](https://github.com/user-attachments/assets/93308080-edbb-4c21-9a2d-23955c061af8)

Am I missing something?

Best regards,
Andreas

### References
[1] Gallery of examples, Documentation of [scikit-fem](https://scikit-fem.readthedocs.io/), [example 1](https://scikit-fem.readthedocs.io/en/stable/listofexamples.html#example-1-poisson-equation-with-unit-load).

**Comment by koehlerson** [comment, marked as answer]
https://www.dealii.org/current/doxygen/deal.II/step_3.html

Should be the same BVP

**Reply by adtzlr** [reply]
Hi @koehlerson,

thanks for the reference! The ferrite.jl example is carried out on the rectangle (-1, 1) while the example from scikit-fem is defined on the rectangle (0, 1). That is the difference I've overlooked.

Best regards,
Andreas
